examples/snippets/data_io/cloud/gcs_upload.py

```diff
@@ -0,0 +1,42 @@
+"""
+An example script to demonstrate the use of the gcs_upload function to upload file,
+to a Google Bucket with multiple diferent parameters
+"""
+
+import tamr_toolbox as tbox
+
+# uploads file to gcs without tarring
+tbox.data_io.cloud.gcs_upload(
+    cloud_client="authenticated_gcs_client",
```

Comment:
I think the object you want here is a `google.cloud.storage.Client`.  I would recommend you include `from google.cloud.storage import Client as GcsClient` at the top, and then create ` gcs_client = Client()` to pass into the function.
Then it would be best to show how the `destination_filepath` does not include the bucket name, by including a comment along the lines of `# upload a local file "my_local_directory/my_file.txt" to "gs://my-bucket/path-to-file"` and then passing the appropriate strings to `destination_filepath` and `bucket_name`.